code/modules/mob/living/simple_animal/slime/powers.dm

```diff
@@ -132,6 +132,24 @@
 			"This subject does not have life energy", "This subject is empty", \
 			"I am not satisified", "I can not feed from this subject", \
 			"I do not feel nourished", "This subject is not food")]!</span>")
+		var/mob/living/carbon/carbon_victim = buckled
+		if(istype(carbon_victim))
+			var/should_apply_effect = TRUE
+
+			if(ishuman(carbon_victim))
+				var/mob/living/carbon/human/lucky_owner = carbon_victim
+				var/head_rating = istype(lucky_owner.head) ? lucky_owner.head.get_armor_rating(BIO) : 0
+				var/suit_rating = istype(lucky_owner.wear_suit) ? lucky_owner.wear_suit.get_armor_rating(BIO) : 0
+				var/uniform_rating = istype(lucky_owner.w_uniform) ? lucky_owner.w_uniform.get_armor_rating(BIO) : 0
```

Comment:
- You can handle all of this in your status effect `on_apply`
- There is more than likely helpers to do this kind of check for you, like `run_armor_check`

Reply:
I didn't want to handle this in `on_apply` as to make the status effect as reusable as possible, it would be pretty annoying to use if it applied by random chance depending on the armor the person is wearing

Reply:
Fair enough, though it can definitely be cleaned up a bit